### PR TITLE
redirect to newly created lyrics

### PR DESCRIPTION
**Pull request by kkuepper**

### CODE REVIEW

**Review by sifferhans — APPROVED** [review]

**Comment by sifferhans** [comment]
https://github.com/user-attachments/assets/b80635e0-34ea-453b-af6c-9a2e9aad7196

**Comment by sifferhans** [comment]
Seems like it now requires an ID to be passed into the request?
CleanShot 2024-12-03 at 00 39 55@2x

File: pages/lyrics/index.vue [hunk 1]
```diff
@@ -26,14 +27,15 @@ async function createLyrics() {
   loading.value = true;
 
   try {
-    await new LyricsApi().lyricsPost({
+    const newId = await new LyricsApi().lyricsPost({
```
Comment (sifferhans) [inline pages/lyrics/index.vue:30]: We dont really seem to get an id back here

File: pages/lyrics/index.vue [hunk 2]
```diff
@@ -26,14 +27,15 @@ async function createLyrics() {
...
       lyrics: {
         songTitle: createForm.title,
         source: "Manual",
         longCopyright: DEFAULT_LONG_COPYRIGHT,
       },
```
Comment (sifferhans) [inline pages/lyrics/index.vue:35]: We now get an error saying `id` is required here

**Review by sifferhans — CHANGES_REQUESTED** [review]

File: pages/lyrics/index.vue (hunk 1 above)
Comment (kkuepper) [reply pages/lyrics/index.vue:30]: The API changes weren't deployed to Prod yet.

**Review by sifferhans — APPROVED** [review]

**Pull request merged**